CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.7.5
+- Added changed semantics version in build.gradle
```

Comment:
Perhaps we could mention that the new semantic libraraty (0.2.0) supports all Eiffel events in the eiffel giyhub repo on topic-4-branch.